helm/harbor-scanner-aqua/templates/deployment.yaml

```diff
@@ -16,14 +16,29 @@ spec:
         app.kubernetes.io/name: {{ include "harbor-scanner-aqua.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
     spec:
+      imagePullSecrets:
+        - name: {{ include "harbor-scanner-aqua.fullname" . }}-registry
+      securityContext:
+        fsGroup: 1000
+        runAsUser: 1000
+        runAsNonRoot: true
+      initContainers:
+        - name: scannercli
+          image: {{ template "harbor-scanner-aqua.scannerImageRef" . }}
```

Comment:
Do you think we should also have an `imagePullPolicy` for the scanner?

Reply:
Yes. We can set it explicitly and make configurable with a Helm value.